port/common/omrmem32helpers.c

```diff
@@ -465,12 +470,12 @@ ensure_capacity32(struct OMRPortLibrary *portLibrary, uintptr_t byteAmount)
 {
 	J9HeapWrapper *heapWrapperCursor = NULL;
 	uintptr_t returnValue = OMRPORT_ENSURE_CAPACITY_FAILED;
-#if defined(OMR_ENV_DATA64)
-	/* For 64 bit os, use flag OMRPORT_VMEM_ALLOC_QUICK as it is in the startup period. */
+#if defined(OMR_ENV_DATA64) && defined(LINUX)
+	/* For 64 bit linux, use flag OMRPORT_VMEM_ALLOC_QUICK as it is in the startup period. */
```

Comment:
First letter of `Linux` should be capitalized everywhere
```suggestion
	/* For 64 bit Linux, use flag OMRPORT_VMEM_ALLOC_QUICK as it is in the startup period. */
```